src/components/Dropdown.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react'
+import { oneOf, oneOfType, func, string, number } from 'prop-types'
+import glamorous from 'glamorous'
+import posed from 'react-pose'
+
+import {
+  spacing,
+  colors,
+  radii,
+  fontSizes,
+  lineHeights,
+  shadows,
+  transition,
+} from '../theme'
+import { joinSpacing, toAlpha } from '../utils/style'
+
+const MenuContainer = glamorous.div({
+  position: 'relative',
+  display: 'inline-block',
+})
+
+const PosedMenu = posed.div({
+  open: {
```

Comment:
dang, this seems pretty simple. 👍

Reply:
Maybe we should rename this `MenuTransition` or something similar to make it more clear what is actually happening.